plugin/acl/setup.go

```diff
@@ -0,0 +1,188 @@
+package acl
+
+import (
+	"net"
+	"strings"
+
+	"github.com/coredns/coredns/core/dnsserver"
+	"github.com/coredns/coredns/plugin"
+	"github.com/coredns/coredns/plugin/metrics"
+
+	"github.com/caddyserver/caddy"
+	"github.com/infobloxopen/go-trees/iptree"
+	"github.com/miekg/dns"
+)
+
+const (
+	// QtypeAll is used to match any kinds of DNS records type.
+	// NOTE: The value of QtypeAll should be different with other QTYPEs defined in miekg/dns.
+	QtypeAll uint16 = 299
```

Comment:
You probably want to handle this differently, I think you can use qtype = 0 to for this use-case.

To go around this differently: why do you think 299 is a valid number (considering that new RR types are being assigned code points all the time) ?

Reply:
I thought `0` (`dns.TypeNone`) might have some special use. 299 here is a random number and I just put it on for review. Will change to 0. Thanks.